test/output/out.spec.txt

```diff
@@ -80,6 +80,13 @@ cypress/integration/happyFlow.spec.js:
 
 
 
+cypress/integration/printLogsSuccess.spec.js:
```

Comment:
I would actually expect to have two version of this file now: out.spec.always.txt out.spec.onFail.txt. Same for the other variants. Also make sure there is a spec that checks that when configured with never no files get created infact.